src/app/accounts/accounts.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { AccountsComponent } from "./accounts.component";
+
+import { MaterialComponentsModule } from "../material/material.module";
+import { ButtonMoreModule } from "../button-more/button-more.module";
+
+@NgModule({
+  imports: [CommonModule, MaterialComponentsModule, ButtonMoreModule],
```

Comment:
we do not need separate module for button more -- you can extract it into `shared` module